xscen/config.py

```diff
@@ -6,15 +6,12 @@
 The ``CONFIG`` dictionary contains all values, structured by submodules and functions. For example,
 for function ``function`` defined in ``module.py`` of this package, the config would look like:
 
-.. code-block:: json
-
-    {
-        'module': {
-            'function' : {
-                ... kwargs ...
-             }
-        }
-    }
+.. code-block::
```

Comment:
```suggestion
.. code-block:: yaml
```
Does this exist?

Reply:
It should be: https://pygments.org/docs/lexers/#pygments.lexers.data.YamlLexer

Reply:
ça existe et c'est ajouté!